package.json

```diff
@@ -6,13 +6,18 @@
   "scripts": {
     "test:lint": "eslint ./",
     "test:unit": "jest",
-    "test": "npm run test:unit && npm run test:lint"
+    "test": "npm run test:unit",
+    "test:git-history": "commitlint --from origin/master --to HEAD",
+    "posttest": "npm run test:git-history && npm run test:lint"
   },
   "jest": {
     "preset": "amex-jest-preset",
     "testPathIgnorePatterns": [
       "/node_modules/",
```

Comment:
Interesting that this ignores `node_modules` here.  I don't think we normally have to do that.

Reply:
If you override `testPathIgnorePatterns` it removes the default setting of ignoring `node_modules`. @JamesSingleton can you validate if that is still the case? If it isnt then maybe it can be removed from here.